src/PartielsPy/Partiels.py

```diff
@@ -0,0 +1,49 @@
+"""A main class for Partiels Wrapper"""
+
+import os
+import platform
+import shutil
+
+
+class Partiels:
+    def __init__(self):
+        self.__executable_path = self.__findExecutablePath()
+
+    def __findExecutablePath(self):
+        name = "Partiels"
+        system = platform.system()
+        path = os.environ.get("PARTIELS_PATH")
+        if path:
+            return path
+        path = shutil.which(name)
+        if path:
+            return path
+        if system == "Linux":
+            paths = [
+                os.path.join(os.environ.get("HOME"), "opt", name),
+                os.path.join("/opt/", name),
+            ]
+        elif system == "Windows":
+            architecture = platform.architecture()[0][:2]
+            paths = [
+                os.path.join("autopf" + architecture, name + ".exe"),
```

Comment:
I think it comes from InnoSetup, but I'm not sure it's relevant here. You should use the equivalent of os.environ["ProgramFiles"] or os.environ["ProgramFiles(x86)"]